examples/jsm/postprocessing/SSAARenderPass.js

```diff
@@ -31,6 +31,8 @@ class SSAARenderPass extends Pass {
 		this.sampleLevel = 4; // specified as n, where the number of samples is 2^n, so sampleLevel = 4, is 2^4 samples, 16.
 		this.unbiased = true;
 
+		this.stencilBufferEnabled = stencilBufferEnabled;
```

Comment:
How about we create `sampleRenderTarget` in the constructor? Public access is then safe and you can configure the stencil buffer without the need for a separate target.

Reply:
Ah wait, that does not work since we can't access the dimensions at that point.

In this case, I suggest you just add a property `stencilBuffer` but without changing the ctor signature.

Reply:
I think that looks better. Thank you for your suggestion. I pushed the changes.